### PR TITLE
Riveting

### PR DESCRIPTION
**Pull request by joelrebel**
#### What does this PR do

- Move to `rivets/condition.Condition` type
- Move to `rivets/kv.StatusValue` type
- Update Swagger docs for condition create with added examples.

### CODE REVIEW

File: docs/swagger.json [hunk 1]
```diff
@@ -79,15 +79,14 @@
                 }
             },
             "post": {
-                "description": "Creates a condition on a server",
+                "description": "Creates a condition on a server\nSample firmwareInstall payload, response: https://github.com/metal-toolbox/conditionorc/blob/main/sample/firmwareInstall.md",
```
Comment (DoctorVin) [inline docs/swagger.json:82]: This comment likely belongs in the README as an explicit pointer to developers to an example of how things work.

File: internal/app/config.go [hunk 2]
```diff
@@ -8,7 +8,7 @@ import (
 
 	"github.com/metal-toolbox/conditionorc/internal/model"
 	"github.com/metal-toolbox/conditionorc/internal/orchestrator/notify"
-	ptypes "github.com/metal-toolbox/conditionorc/pkg/types"
+	condition "github.com/metal-toolbox/rivets/condition"
```
Comment (DoctorVin) [inline internal/app/config.go:11]: you don't need the import alias here.

File: internal/orchestrator/orchestrator.go [hunk 3]
```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/metal-toolbox/conditionorc/internal/store"
 	"github.com/metal-toolbox/conditionorc/internal/version"
 	v1EventHandlers "github.com/metal-toolbox/conditionorc/pkg/api/v1/events"
-	ptypes "github.com/metal-toolbox/conditionorc/pkg/types"
+	condition "github.com/metal-toolbox/rivets/condition"
```
Comment (DoctorVin) [inline internal/orchestrator/orchestrator.go:12]: `#no gods, no import aliases`

**Review by DoctorVin — APPROVED** [review]
Content is fine

File: README.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ ConditionOrc provides the [conditions](https://github.com/metal-toolbox/architec
 for consumers wanting to execute actions on server hardware.
 
 It does this by,
- - Providing a CRUD API to setting conditions like `firmwareInstall`, `inventoryOutofband` on a server.
+ - Providing a CRUD API to setting conditions like `firmwareInstall`, `Inventory` on a server.
```
[the remark this answers is not in the document]
Comment (joelrebel) [reply README.md]: that was a typo, thanks!

File: docs/docs.go [hunk 5]
```diff
@@ -196,7 +195,7 @@ const docTemplate = `{
         }
     },
     "definitions": {
-        "types.Condition": {
+        "condition.Condition": {
```
[the remark this answers is not in the document]
Comment (joelrebel) [reply docs/docs.go:198]: fixed

File: docs/swagger.json (hunk 1 above)
Comment (joelrebel) [reply docs/swagger.json:82]: Added it to the README as well. I'd like devs referring to the OpenAPI spec to find it as well.

File: internal/orchestrator/orchestrator.go (hunk 3 above)
Comment (joelrebel) [reply internal/orchestrator/orchestrator.go]: fixed

File: internal/app/config.go (hunk 2 above)
Comment (joelrebel) [reply internal/app/config.go]: fixed

File: internal/store/serverservice.go [hunk 6]
```diff
@@ -43,7 +43,7 @@ var (
 	ErrServerserviceAttribute = errors.New("error in serverservice attribute")
 
 	// ServerserviceConditionsNSFmtStr attribute namespace format string value for server condition attributes.
-	ServerserviceConditionsNSFmtStr = "sh.hollow.condition.%s"
+	ServerserviceConditionsNSFmtStr = "sh.hollow.rctypes.%s"
```
Comment (DoctorVin) [inline internal/store/serverservice.go:46]: The name of the import should not be leaking into the NATS subject name. An outside observer expects the “thing” we’re referring to as a “condition”. The token we use to import this is an internal implementation detail.

**Review by jakeschuurmans — DISMISSED** [review]

File: internal/store/serverservice.go (hunk 6 above)
Comment (joelrebel) [reply internal/store/serverservice.go]: thanks, fixed

**Review by DoctorVin — APPROVED** [review]

**Pull request merged**